newsletter-october-2021.rst

```diff
@@ -0,0 +1,98 @@
+.. post:: October 6, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in October.
+
+Read the Docs newsletter - October 2021
+=======================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We have resumed sending our blog updates by email.
+  You can `subscribe to our newsletter <https://landing.mailerlite.com/webforms/landing/p8b7z2>`_
+  so you don't miss them.
+- Version 3 of our Embed API is now implemented,
+  thanks to :doc:`the CZI grant we received</czi-grant-announcement>`.
+  We will give more details about it soon.
+- We have reworked :doc:`our team page <readthedocs:team>`
+  to give you a better idea of who we are and how we work at the company.
+
+New features
+------------
```

Comment:
New things missing here:

- https://github.com/readthedocs/readthedocs.org/pull/8498
- https://github.com/readthedocs/readthedocs.org/pull/8477

Reply:
Right, thanks! I missed that those were user-facing. I will include a screenshot of the latter.